ext-apiserver/pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -32,6 +30,9 @@ import (
 	"k8s.io/kube-deploy/ext-apiserver/pkg/apis/cluster/common"
 )
 
+// Finalizer is set on PreareForCreate callback
+const MachineFinalizer string = "MachineFinalizer"
```

Comment:
This should be tagged with our API domain. Like `machine.cluster.k8s.io` or something like that.